drake/systems/bot_visualizer_system.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include <lcm/lcm-cpp.hpp>
+
+#include "drake/drakeBotVisualizerSystem_export.h"
+#include "drake/lcmt_viewer_draw.hpp"
+#include "drake/systems/framework/context.h"
+#include "drake/systems/framework/leaf_system.h"
+#include "drake/systems/plants/RigidBodyTree.h"
+
+namespace drake {
+namespace systems {
+
+/**
+ * This is a Drake System 2.0 block that communicates with the Drake Visualizer
+ * via LCM messages. It does this in two phases: initialization, which runs once
+ * at startup, and run-time, which periodically runs for the duration of the
+ * simulation.
+ *
+ * During initialization, this system block analyzes the RigidBodyTree and
+ * tells Drake Visualizer what it will be visualizing. For example, these
+ * include the number of rigid bodies and their dimensions. The LCM message used
+ * during this phase is called `drake::lcmt_viewer_load_robot` and the channel
+ * name is "DRAKE_VIEWER_LOAD_ROBOT".
+ *
+ * During run-time, this system block takes the current state of the model and
+ * tells Drake Visualizer how to visualize the model. For example, this includes
+ * the position and orientation of each rigid body mentioned during
+ * initialization. The LCM messages used during this phase is
+ * `drake::lcmt_viewer_draw` and the channel name is "DRAKE_VIEWER_DRAW".
+ *
+ * To support multiple simultaneous instances of this system, which may
+ * occur when multiple unit tests are running in parallel, the channel names
+ * can be modified by a postfix string, which is specified as an input parameter
+ * to the constructor.
```

Comment:
please add @ingroup systems to make it show up here:
http://drake.mit.edu/doxygen_cxx/group__systems.html
example here: https://github.com/RobotLocomotion/drake/blob/master/drake/systems/framework/primitives/adder.h#L16
@amcastro-tri -- would be good to do this for RigidBodyPlant, too